src/components/Footer.js

```diff
@@ -1,16 +1,47 @@
-import React from 'react'
+import React from 'react';
+import { FaFacebookSquare, FaTwitterSquare, FaLinkedin, FaEnvelope } from 'react-icons/fa';
```

Comment:
Could you make the icons responsive and a little smaller?

Reply:
In the most recent commit I have added "flex" to the social media icons to make them responsive. Also, have changed the size of the icons from 50 down to 40, let me know if they still need to be smaller.